packages/invoices/src/OutboundShipment/DetailView/OutboundLineEdit/hooks/useNextItem.test.tsx

```diff
@@ -205,11 +205,9 @@ describe('useNextItem', () => {
       wrapper: TestingProvider,
     });
 
-    return result.waitForNextUpdate().then(() => {
-      expect(result.result.current.next).toEqual(null);
-      expect(result.result.current).toEqual(
-        expect.objectContaining({ disabled: true })
-      );
-    });
+    expect(result.result.current.next).toEqual(null);
```

Comment:
not sure about this one! it wasn't re-rendering, so didn't require the wait anymore.. 🤷‍♂️